docs/src/main/sphinx/connector/phoenix.md

```diff
@@ -283,4 +283,4 @@ statements, the connector supports the following features:
 ### Procedures
 
 ```{include} procedures-execute.fragment
-```
\ No newline at end of file
+```
```

Comment:
@mosabua stupid question: how... is it related to idea config? how to revert it, on mylocal the line seems identicial to other connectors .md file...

Reply:
not sure what is going on here. Sometimes github just gets its diff confused. As long as the build works .. this is fine.

Reply:
I did do some refactoring in that docs section recently .. maybe it came up as a result from the rebase .. but essentially.. dont worry.